crypto/aes/asm/aesv8-armx.pl

```diff
@@ -110,15 +110,15 @@
 # This file generates .s file for 64-bit and 32-bit CPUs.
 # We don't implement .rodata on 32-bit CPUs yet.
 #
-$code.=".rodata\n"	if ($flavour =~ /64/);
+$code.=".rodata\n"	if ($flavour =~ /.*64/);
```

Comment:
This makes no change to which strings are matched or not (`.*` means "zero or more characters")

Reply:
Since `$flavour` is `win64` for llvm-mingw aarch64, how about changing `if ($flavour =~ /64/);` to `if ($flavour =~ /64/) && ($flavour ne "win64");` in both these lines?

I don't know what other platforms would have `$flavour` as `win64` and whether this change would be a problem for them